index.src.html

```diff
@@ -650,6 +655,17 @@ <h4 id="html-iframe-element-attributes">Attributes</h4>
   The <code>referrerPolicy</code> IDL attribute must <a>reflect</a> the
   <code>referrerpolicy</code> content attribute, <a>limited to only known
   values</a>.
+
+  <h3 id="html-link-element">HTMLLinkElement</h3>
+  <pre class="idl">
+    partial interface HTMLLinkElement {
+      attribute DOMString referrerPolicy;
+    };
+  </pre>
+  <h4 id="html-link-element-attributes">Attributes</h4>
+  The <code>referrerPolicy</code> IDL attribute must <a>reflect</a> the
+  <code>referrerpolicy</code> content attribute, <a>limited to only known
+  values</a>.
```

Comment:
would you expect a bookmark to remember the referrer for its future navigations?

anyway, I guess the best we can do is saying something like if this link elements triggers a fetch or a navigation that creates a referrer from the current document, the referrer policy should apply.